build_library/oem_aci_util.sh

```diff
@@ -0,0 +1,80 @@
+# Copyright (c) 2016 The CoreOS Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# Expects BOARD, BUILD_DIR, BUILD_LIBRARY_DIR, and COREOS_VERSION in env.
+
+# There must be a manifest template included with the ebuild at
+# files/manifest.in, which will have some variable values substituted before
+# being written into place for the ACI.  Optionally, a shell script can also be
+# included at files/manglefs.sh to be run after all packages are installed.  It
+# is intended to be used to make modifications to the file system layout and
+# program paths that some included agent software might expect.
+
+. "${BUILD_LIBRARY_DIR}/prod_image_util.sh"
+
+oem_aci_create() {
+    local aciroot="${BUILD_DIR}"
+    local oem="${1?No OEM was specified}"
+    local base_pkg="coreos-base/coreos-oem-${oem}"
+    local ebuild=$(equery-"${BOARD}" w "${base_pkg}" 2>/dev/null)
+    local staging_image="coreos_oem_${oem}_aci_stage.bin"
+
+    [ -n "${ebuild}" ] || die_notrace "No ebuild exists for OEM \"${oem}\""
+    grep -Fqs '(meta package)' "${ebuild}" ||
+        die_notrace "The \"${base_pkg}\" ebuild is not a meta package"
+
+    # Disable some features, and run a regular prod image build for this OEM.
+    # Note that we later depend on this function using "${BUILD_DIR}/rootfs".
+    local save_COREOS_OFFICIAL=$COREOS_OFFICIAL
+    local save_FLAGS_upload=$FLAGS_upload
+    COREOS_OFFICIAL=0
+    FLAGS_upload=1
+    mkdir -p "${BUILD_DIR}"/configroot/etc/portage/
+    echo 'dev-lang/python' >> "${BUILD_DIR}"/configroot/etc/portage/package.unmask
```

Comment:
I don't think we should be using `dev`, as it would leave so much unused bloat in the container (static libraries, build tools, etc.).  If unmasking a package is unacceptable here, we should have another sub-profile that doesn't mask packages.